packages/plan-form-core/src/formHelpers/constants/stringConstants.tsx

```diff
@@ -0,0 +1,140 @@
+export const TRIGGER = 'trigger';
+
+// plan activity codes
+export const CASE_CONFIRMATION_ACTIVITY_CODE = 'caseConfirmation';
+export const FAMILY_REGISTRATION_ACTIVITY_CODE = 'familyRegistration';
+export const BLOOD_SCREENING_ACTIVITY_CODE = 'bloodScreening';
+export const BEDNET_DISTRIBUTION_ACTIVITY_CODE = 'bednetDistribution';
+export const LARVAL_DIPPING_ACTIVITY_CODE = 'larvalDipping';
+export const MOSQUITO_COLLECTION_ACTIVITY_CODE = 'mosquitoCollection';
+export const BCC_ACTIVITY_CODE = 'BCC';
+export const IRS_ACTIVITY_CODE = 'IRS';
+export const MDA_POINT_DISPENSE_ACTIVITY_CODE = 'pointDispenseMDA';
+export const MDA_POINT_ADVERSE_EFFECTS_ACTIVITY_CODE = 'pointAdverseMDA';
+export const DYNAMIC_FAMILY_REGISTRATION_ACTIVITY_CODE = 'dynamicFamilyRegistration';
+export const DYNAMIC_BLOOD_SCREENING_ACTIVITY_CODE = 'dynamicBloodScreening';
+export const DYNAMIC_BEDNET_DISTRIBUTION_ACTIVITY_CODE = 'dynamicBednetDistribution';
+export const DYNAMIC_LARVAL_DIPPING_ACTIVITY_CODE = 'dynamicLarvalDipping';
+export const DYNAMIC_MOSQUITO_COLLECTION_ACTIVITY_CODE = 'dynamicMosquitoCollection';
+export const DYNAMIC_BCC_ACTIVITY_CODE = 'dynamicBCC';
+export const DYNAMIC_IRS_ACTIVITY_CODE = 'dynamicIRS';
+export const DYNAMIC_MDA_COMMUNITY_DISPENSE_ACTIVITY_CODE = 'dynamicCommunityDispenseMDA';
+export const DYNAMIC_MDA_COMMUNITY_ADHERENCE_ACTIVITY_CODE = 'dynamicCommunityAdherenceMDA';
+export const PRODUCT_CHECK_ACTIVITY_CODE = 'dynamicProductCheck';
+export const FIX_PRODUCT_PROBLEM_ACTIVITY_CODE = 'dynamicFixProductProblem';
+export const RECORD_GPS_ACTIVITY_CODE = 'dynamicRecordGPS';
+export const SERVICE_POINT_CHECK_ACTIVITY_CODE = 'dynamicServicePoint';
+
+// task action codes
+export const BCC_CODE = 'BCC';
+export const IRS_CODE = 'IRS';
+export const PRODUCT_CHECK_CODE = 'Product Check';
+export const FIX_PRODUCT_PROBLEMS_CODE = 'Fix Product Problems';
+export const RECORD_GPS_CODE = 'Record GPS';
+export const SERVICE_POINT_CHECK_CODE = 'Service Point Check';
+export const BEDNET_DISTRIBUTION_CODE = 'Bednet Distribution';
+export const BLOOD_SCREENING_CODE = 'Blood Screening';
+export const CASE_CONFIRMATION_CODE = 'Case Confirmation';
+export const RACD_REGISTER_FAMILY_CODE = 'RACD Register Family';
+export const LARVAL_DIPPING_CODE = 'Larval Dipping';
+export const MOSQUITO_COLLECTION_CODE = 'Mosquito Collection';
+export const MDA_POINT_DISPENSE_CODE = 'MDA Dispense';
+export const MDA_POINT_ADVERSE_EFFECTS_CODE = 'MDA Adverse Event(s)';
+export const MDA_ADHERENCE_CODE = 'MDA Adherence';
+export const GOAL_CONFIRMATION_GOAL_ID = 'Case_Confirmation';
+export const GOAL_ID = 'goal_id';
+
+export const CREATE_TYPE = 'create';
+
+// dynamic plan activities
+export const NAMED_EVENT_TRIGGER_TYPE = 'named-event';
+export const PLAN_ACTIVATION_TRIGGER_NAME = 'plan-activation';
+export const APPLICABILITY_CONDITION_KIND = 'applicability';
+
+// planform fields
+export const activities = 'activities';
```

Comment:
Do we need the type defs in this section?

Reply:
yeah, so that I can use them as computed types in the `planFormFieldsValue` interface.
They enable for me to do sth like
```typescript
const property = 'firstName';
type property = typeof property;

interface Person{
[property] : string
}
```
which translates to :
```typescript
interface Person{
firstName: string
}
```

Reply:
Cool